includes/cli/class-wc-cli-tracker-command.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * WC_CLI_Tracker_Command class file.
+ *
+ * @package WooCommerce\CLI
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Allows access to tracker snapshot for transparency and debugging.
+ *
+ * @since x.x.x
+ * @package WooCommerce
+ */
+class WC_CLI_Tracker_Command {
+
+	/**
+	 * Registers a command for showing WooCommerce Tracker snapshot data.
+	 */
+	public static function register_commands() {
+		WP_CLI::add_command( 'wc tracker-snapshot', array( 'WC_CLI_Tracker_Command', 'show_tracker_snapshot' ) );
+	}
+
+	/**
+	 * Dump tracker snapshot data to screen.
+	 */
+	public static function show_tracker_snapshot() {
+		$snapshot_data = WC_Tracker::get_tracking_data();
+
+		// Using print_r as a convenient way to dump the snapshot data to screen in a readable form.
+		WP_CLI::log( print_r( $snapshot_data, true ) ); // phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_print_r
+		// Could improve this, e.g. flatten keys (join nested fields with `-`) and render as CSV or json.
```

Comment:
This is super dumb/basic for now. If there's interest we can do more work formatting the data, but I don't think that's super essential right now – the key is to make it possible to view the snapshot on demand (without hacking source code).

Reply:
It's possible to format it with just this lines of code:

```php
	public static function show_tracker_snapshot( $args, $assoc_args ) {
		$snapshot_data = WC_Tracker::get_tracking_data();

		$formatter = new \WP_CLI\Formatter(
			$assoc_args,
			array_keys( $snapshot_data )
		);

		$formatter->display_items( array( $snapshot_data ) );
	}
```

It will display a table by default, and also allow commands like `wp wc tracker snapshot --format=json` and `wp wc tracker snapshot --format=csv`, also allows using filters like `wp wc tracker snapshot --format=json --field=wp` or with many fields `wp wc tracker snapshot --format=json --fields=url,email,wp`

But note that there's a lot of information for just one table, even with flatten keys will be very hard to display everything in just one table.

Reply:
This is much more flexible! I tested and `json` format works well, as does `yaml` (exceptionally readable and compact). The other column-based formats are not really practical, but might work better for some uses 🤷 

I've switched to this approach and added info & examples to the docs here: 414ba48ee230248fc77d5e35769eca1a1e87cefa